.circleci/config.yml

```diff
@@ -490,12 +491,21 @@ jobs:
       - checkout
       - build/determinator:
           <<: *all_gateways_build_verify
+      - docker/install-dc
       - python/set_version
       - run:
           name: Lint cloud Go code
           command: |
             cd ${MAGMA_ROOT}/orc8r/cloud/docker
             python3 build.py --lint
+      - run:
+          name: Generate test coverage
+          command: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/docker
+            python3 build.py --coverage
+      - run:
```

Comment:
I think @bbarritt had a CircleCI orb in order to do that.

Reply:
The orb you're referencing is a third-party orb, which means we need to explicitly allow third-party orbs

If you want to go ahead and enable that for our project then I can switch over to using the orb 🙂

Reply:
I already enabled it!